<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAccDialog</class>
 <widget class="QDialog" name="AddAccDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Account</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(40, 40, 40);</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="ScanQrButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>30</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(1, 0, 0);
border-width: 1px;
border-style: solid;
border-color: rgb(0, 255, 0);
color: rgb(0, 255, 0);
text-align:center;</string>
   </property>
   <property name="text">
    <string>Scan QR</string>
   </property>
  </widget>
  <widget class="QLabel" name="QR_instructions">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>220</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Make sure QR code is fully visible on main screen</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterManuallyButton">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>210</y>
     <width>200</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background-color: rgb(1, 0, 0);
border-width: 1px;
border-style: solid;
border-color: rgb(0, 255, 0);
color: rgb(0, 255, 0);
text-align:center;</string>
   </property>
   <property name="text">
    <string>Enter Manually</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
